multi_map_it_test.go

```diff
@@ -450,3 +450,120 @@ func TestMultiMap_TryLockWithLeaseAndTimeout(t *testing.T) {
 		assert.False(t, b)
 	})
 }
+
+func TestMultiMap_ContainsKey(t *testing.T) {
+	it.MultiMapTester(t, func(t *testing.T, m *hz.MultiMap) {
+		ctx := context.Background()
+		keyExists, err := m.ContainsKey(ctx, "key")
+		if err != nil {
+			t.Fatal(err)
+		}
+		assert.False(t, keyExists)
+		// assert key value pair
+		it.MustValue(m.Put(ctx, "key", "value"))
+		keyExists, err = m.ContainsKey(ctx, "key")
+		if err != nil {
+			t.Fatal(err)
+		}
+		assert.True(t, keyExists)
+	})
+}
+
+func TestMultiMap_ContainsValue(t *testing.T) {
+	it.MultiMapTester(t, func(t *testing.T, m *hz.MultiMap) {
+		ctx := context.Background()
+		valueExists, err := m.ContainsValue(ctx, "value")
```

Comment:
How about `exists` instead of valueExists?

Reply:
Thanks, fixed it at c1204cb